### DISCUSSION TITLE
Long loading/responses after reinstallation.

### DISCUSSION THREAD

**Discussion by ArZeroNero** [oobabooga/text-generation-webui#6457] [Q&A]
Briefly - before reinstallation of win 11, I had a AI working quietly, it generated answers quickly and it was a sin to complain, everything was fine.

I reinstalled, but left the files in the same state.

It started to load much longer, and besides, if my big answers were generated in maybe 33 seconds, now small answer GENERATES ONE EVOLUTION PROMPT 400 SECONDS, ONE!!! Before that, he had 10 times in 33 seconds.

I'm sorry, I'm just bombing. Who knows a possible solution and what's missing? Except, of course, brains.

I tried reinstalling the WebUI, didn't help.

**Comment by tailerDerdon** [comment]
You may currently be using a CPU, but previously used a graphics card. You may have previously installed drivers and Cuda. But I could be wrong. This is the only more or less understandable normal library that I managed to run using the GPU. If you just moved files, then your connections have been broken, in this case, just reinstall everything from the very beginning as written in the instructions.

**Reply by ArZeroNero** [reply]
I just reinstalled it again and... it worked??? so eh... nevermind!
